### ISSUE TITLE
Dataproc - Unable to upgrade to 5.4.0 libraries-bom

### ISSUE THREAD

**Issue by lesv** [GoogleCloudPlatform/java-docs-samples#2942]
Found in #2937
https://source.cloud.google.com/results/invocations/f7079bef-8021-4a51-b88a-24740f041202/targets/cloud-devrel%2Fjava%2Fjava-docs-samples%2Fjava11%2Fpresubmit/log

# Java 11
```
------------------------------------------------------------
- testing dataproc
------------------------------------------------------------

[ERROR] Tests run: 1, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 299.771 s <<< FAILURE! - in InstantiateInlineWorkflowTemplateTest
[ERROR] InstantiateInlineWorkflowTemplateTest.instanstiateInlineWorkflowTest  Time elapsed: 299.769 s  <<< ERROR!
java.util.concurrent.CancellationException: Task was cancelled.
	at com.google.common.util.concurrent.AbstractFuture.cancellationExceptionWithCause(AbstractFuture.java:1388)
	at com.google.common.util.concurrent.AbstractFuture.getDoneValue(AbstractFuture.java:562)
	at com.google.common.util.concurrent.AbstractFuture.get(AbstractFuture.java:545)
	at com.google.common.util.concurrent.FluentFuture$TrustedFuture.get(FluentFuture.java:86)
	at com.google.common.util.concurrent.ForwardingFuture.get(ForwardingFuture.java:62)
	at com.google.api.gax.longrunning.OperationFutureImpl.get(OperationFutureImpl.java:127)
	at InstantiateInlineWorkflowTemplate.instantiateInlineWorkflowTemplate(InstantiateInlineWorkflowTemplate.java:111)
	at InstantiateInlineWorkflowTemplateTest.instanstiateInlineWorkflowTest(InstantiateInlineWorkflowTemplateTest.java:59)

```

@bradmiro @stephaniewang526 PTAL

**Comment by bradmiro** [comment]
@chingor13 are you familiar with any issues regarding LROs timing out at five minutes? I'm able to repeat this error consistently only when using the Java client library.

**Comment by bradmiro** [comment]
Seems to be a breaking change as of Dataproc 0.121.0

**Comment by bradmiro** [comment]
Still working on this...

**Comment by bradmiro** [comment]
Should be fixed in the next release.

**Comment by lesv** [comment]
@bradmiro Can you request an update to GoogleAPIs/google-cloud-bom to the correct version so this get's fixed?

**Comment by lesv** [comment]
@elharo FYI so you can follow the issue.

**Comment by lesv** [comment]
@bradmiro It's actually googleapis/java-cloud-bom

**Comment by meredithslota** [comment]
https://github.com/GoogleCloudPlatform/java-docs-samples/pull/2937 looks merged — is this fixed?

**Comment by lesv** [comment]
Appears to have been fixed in #3187 

Thank you everyone!
